package.json

```diff
@@ -22,7 +22,7 @@
   "homepage": "https://iot.mozilla.org",
   "dependencies": {
     "asn1.js": "^4.9.1",
-    "bcrypt-nodejs": "0.0.3",
+    "bcryptjs": "^2.4.3",
```

Comment:
3> much needed... We need some process around this too see https://github.com/nodesecurity/nsp